docs/operator-manual/rbac.md

```diff
@@ -212,3 +212,29 @@ Or against the group defined:
 $ argocd admin settings rbac can db-admins get applications 'staging-db-admins/*' --policy-file policy.csv
 Yes
 ```
+
+### Adding additional RBAC configmaps
+Since v2.5, you now have the ability to create additional ConfigMaps to augment the policy specified in `argocd-rbac-cm`.
+
+Simply create a ConfigMap with the label `argocd.argoproj.io/cm-type=additional-rbac` and specify the `policy.csv` key.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-rbac-cm-extra
+  namespace: argocd
+  labels:
+    argocd.argoproj.io/cm-type: additional-rbac
+  policy.csv: |
```

Comment:
policy.csv should be inside data

data:
  policy.default: role:readonly
  policy.csv: |